src/scripts/main.js

```diff
@@ -1,7 +1,27 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.querySelector('body');
+  const message = document.createElement('div');
+  const titleOfMessage = document.createElement('h2');
+  const descriptionOfMessage = document.createElement('p');
```

Comment:
```suggestion
  const messageTitle = document.createElement('h2');
  const messageDescription = document.createElement('p');
```
I think it's better not to use `Of` in variables' names. It makes them a bit harder to read